CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+
+### code directory
+
+The [code](code) directory contains all the files related to code snippets and CI testing.
+Files of particular importance are:
+
+* [CodeTester.cpp](code/CodeTester.cpp): Contains all Fast DDS pub-sub and RTPS layer snippets.
+It is a buildable file, so it can be compiled and linked against the Fast DDS to verify that all samples of code are up to date.
+Furthermore, it is used to create an executable that is then used to generate Ctest tests.
+* [DDSCodeTester.cpp](code/DDSCodeTester.cpp): Contains Fast DDS DDS layer snippets.
+* [XMLTester.xml](code/XMLTester.xml): Contains XML snippets.
+* [doxygen-config.in](code/doxygen-config.in): Doxyfile to configure Doxygen for creating Fast DDS API reference.
+
+## Style guide
+
+This style guide outlines writing directives and dos and don'ts.
+Contributors and reviewers should always refer to them when introducing new content so that the text keeps a consistent form throughout the entire documentation.
+It is the responsibility of the project maintainers to enforce this guide on every contribution, and to keep it up to date.
+The guide is divided in 3 sections:
+
+1. [Writing style](#writing-style)
+1. [Text structure](#text-structure)
+1. [Content](#content)
+
+### Writing style
+
+1. Use direct and concise language.
+Keep in mind that nobody particularly enjoys reading technical documentation, so make your points as clear and fast as possible so the reader can quickly get what they need and move on.
+1. Use reported speech when writing the documentation, as it is the most suitable mode for formal technical English.
+1. In this sense, avoid contractions of any kind.
+1. Direct speech may be used on tutorials, but it should be replaced with imperative mode with no subject when possible.
+1. Always state what an acronym stands for the first time you use it.
+Try to also state it from time to time as you go.
+
+### Text structure
+
+1. Go from the abstract to the concrete.
+Give a big picture first and then start digging into the specifics.
+1. Always write keeping maintainability in mind.
+1. Split you content in different pages as much as readability allows.
+This makes it easier for everyone to find just the piece of information they need without scrolling back and forth over endlessly long pages.
+1. Tables are a great form of comprising a lot of data in a small, easy to consult manner.
+Use them as much as you can, even if it duplicates some information.
+1. Use as many `note`, `important`, and `warning` blocks as you need.
+They are a great way to convey a really important message the reader should not overlook.
+1. Add cross-references to other relevant sections, API reference, and external sites or documents as much as possible.
+Make sure the reader has easy access to all relevant information they may need for fully comprehending the documentation.
```

Comment:
In this context I think it would be *need + to-infinitive*.

```suggestion
Make sure the reader has easy access to all relevant information they may need to fully comprehend the documentation.
```

Reply:
Both work IMHO. I'll take yours